components/clp-package-utils/clp_package_utils/controller.py

```
@@ -247,6 +249,49 @@ def _set_up_env_for_redis(self) -> EnvVarsDict:
+        }
+
+        return env_vars
+
+    def _set_up_env_for_spider_scheduler(self) -> EnvVarsDict:
+        """
+        Sets up environment variables for the Spider database component.
+
+        :return: Dictionary of environment variables necessary to launch the component.
+        """
+        component_name = SPIDER_SCHEDULER_COMPONENT_NAME
+        logger.info(f"Setting up environment for {component_name}...")
```

Ruff (0.14.4)

Logging statement uses f-string

(G004)